src/components/dropdown/daikin-dropdown.ts

```diff
@@ -318,6 +314,66 @@ export class DaikinDropdown extends LitElement {
     }
   }
 
+  private _updateFormValue() {
+    this._internals.setFormValue(JSON.stringify(this.value));
+  }
+
+  private _updateItemsSelectable(): void {
+    for (const item of this._items) {
+      item.selectable = this.multiple;
+    }
+  }
+
+  private _updateValue() {
+    this.value = this.selectedOptions.at(-1) ?? null;
+  }
+
+  private _updateSelectedOptionsByValue(): void {
+    this.selectedOptions = this.value ? [this.value] : [];
+  }
+
+  private _addSelection(value: string): void {
+    this.selectedOptions = this.multiple
+      ? this.selectedOptions.includes(value)
+        ? this.selectedOptions
+        : [...this.selectedOptions, value]
+      : [value];
+  }
+
+  private _removeLastSelection(): void {
+    if (!this.multiple) {
+      this.value = null;
+      this.selectedOptions = [];
+      return;
+    }
+
+    const removeOption = this.selectedOptions.pop();
+    this.selectedOptions = this.selectedOptions.filter(
+      (option) => option != removeOption
+    );
+  }
```

Comment:
We would like to standardize whether to update `value` here together with updating `selectedOptions` or to leave updating `value` to `willUpdate`.
In the latter case, a comment of our intent is required.

Also, note that `.pop()` removes an element from the array so

```ts
    this.selectedOptions = this.selectedOptions.filter(
      (option) => option != removeOption
    );
```

is not needed. ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop

```suggestion
  private _addSelection(value: string): void {
    this.selectedOptions = this.multiple
      ? this.selectedOptions.includes(value)
        ? this.selectedOptions
        : [...this.selectedOptions, value]
      : [value];
    this.value = this.selectedOptions.at(-1) ?? null;
  }

  private _removeLastSelection(): void {
    if (!this.multiple) {
      this.selectedOptions = [];
    } else {
      this.selectedOptions.pop();
    }
    this.value = this.selectedOptions.at(-1) ?? null;
  }
```

Reply:
@yodas7 
```
private _removeLastSelection(): void {
```
In the this line, I think that null is probably in the value, but is it necessary to update this?